src/containers/EventContainer/EventContainer.js

```diff
@@ -3,13 +3,27 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import EventCard from '../../components/EventCard/EventCard';
 
-export const EventContainer = ({ pathname, events }) => {
-  const styles = pathname.includes('profile') ? 'EventContainer profile' : 'EventContainer home';
-    return (
-      <div className={styles}>
-      {
-        events.map(event => 
-          <EventCard 
+export const EventContainer = ({ pathname, events, userEvents }) => {
```

Comment:
I did make some changes in this component so we might have some merge conflicts to fix.